packages/build/src/error/parse/plugin.ts

```diff
@@ -36,27 +36,32 @@ const getVersion = function ({ version }) {
   return version
 }
 
-export const getHomepage = function (pluginPackageJson = {}, { loadedFrom } = {}) {
+type pkgJSONData = { name?: string; bugs?: { url?: string }; repository?: { url?: string } }
+
+export const getHomepage = function (
+  pluginPackageJson: pkgJSONData = {},
+  { loadedFrom }: { loadedFrom?: string } = {},
+) {
   return (
     getRepository(pluginPackageJson) ||
     getNpmLink(pluginPackageJson, { loadedFrom }) ||
     getIssuesLink(pluginPackageJson)
   )
 }
 
-const getRepository = function ({ repository: { url } = {} }) {
+const getRepository = function ({ repository: { url } = {} }: { repository?: { url?: string } }) {
   return url
 }
 
-const getNpmLink = function ({ name }, { loadedFrom }) {
+const getNpmLink = function ({ name }: { name?: string }, { loadedFrom }) {
```

Comment:
Did you mean to also type `loadedFrom`?

Reply:
Done via - https://github.com/netlify/build/pull/5374/commits/0933e2e13d629726e0b25dd9a4dad4dd854cff7e